### ISSUE TITLE
Unable to build: loadTools not working due to broken links (Error 404)

### ISSUE THREAD

**Issue by mapero** [jni4net/jni4net#34]
Hi,
i think you forget to move your binary dependencies from googlecode to github. I tried to build jni4net from source, but it fails because of missing dependencies, e.g. selvin.exportdll-0.2.6.0.dll.
I was not able to get it from anywhere else.

Tried http://jni4net.googlecode.com/svn/mvnrepo/net/sf/jni4net/selvin.exportdll/0.2.6.0/selvin.exportdll-0.2.6.0.dll

Can you reupload or fix the links please.

Thanks.

**Comment by pavelsavara** [comment]
for now try this guide
http://blog.diniscruz.com/2013/08/creating-fork-of-jni4netdealing-with.html

**Comment by mapero** [comment]
Sorry, but I still could not find the files since there is no tools/lib folder on the master, generics-08 or any other branch. I also tried to check commits from the past without success.

**Comment by pavelsavara** [comment]
you are right. I just fixed part of it
- I found some backup of maven plugins and build tool circa year 2011. 
- I was not able to find code nor binaries for selvin 0.2.6.0 either, facepalm.
- i didn't try to build the build tools
- https://github.com/jni4net/build

I updated links in main jni4net repo to point to maven repo on github raw
- https://github.com/jni4net/jni4net.github.io/tree/master/tools/libs
- https://github.com/jni4net/jni4net.github.io/tree/master/mvnrepo
